cl/tests/base.py

```diff
@@ -58,7 +58,6 @@ def _create_browser() -> webdriver.Chrome:
         if settings.SELENIUM_HEADLESS is True:
             options.add_argument("headless")
         options.add_argument("silent")
-        options.add_experimental_option("w3c", False)
```

Comment:
This option isn't supported in the latest selenium, not sure why it was needed but seems we can remove it now, the commit in 65013fdac3414dddce73ce2dc1c702e9442a9f5a adding this flag just says:

> Adds the w3c False flag, which solves some dumb Selenium thing.